docs/source/tutorial.rst

```diff
@@ -294,3 +294,77 @@ The stack of DataPipes can then be constructed using their functional forms (rec
 
 In the above example, ``datapipes1`` and ``datapipes2`` represent the exact same stack of ``IterDataPipe``\s. We
 recommend using the functional form of DataPipes.
+
+Working with Cloud Storage Providers
+---------------------------------------------
```

Comment:
I recommend to link those `DataPipes` with functional API and the corresponding online doc here.